Makefile

```diff
@@ -17,14 +17,14 @@ GIT_VERSION=`echo ${VERSION} | sed -e s/\~//g`
 PI_VERSION=$(VERSION)
```

Comment:
I can not see, what has changed in this file.
Can you please revert it?

Reply:
still nothing to see?
I moved some targets (don't call `clean` when no target is specified) and fixed some white-spaces (make is very TAB-sensitive).